src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Model.cs

```diff
@@ -16,6 +16,9 @@ internal class Model
         public TextSpan TextSpan { get; }
         public IList<SignatureHelpItem> Items { get; }
         public SignatureHelpItem SelectedItem { get; }
+
+        /// <summary>UserSelected is true if the SelectedItem is the result of a user selection (up/down arrows).</summary>
+        public bool UserSelected { get; }
```

Comment:
note: it's also possible to change sig help items with the mouse i believe.  you may want to at least manually test that (to make sure it doesn't crash).

Reply:
Thanks. Tested manually in IDE. Using the mouse (to click on the arrow up/down symbols) seems to work just like regular arrow up/down keys.